cluster/manager/manager.go

```diff
@@ -1887,17 +1894,17 @@ func (c *ClusterManager) SecretGetDefaultSecretKey() (interface{}, error) {
 
 // SecretCheckLogin validates session with secret store
 func (c *ClusterManager) SecretCheckLogin() error {
-	return c.SecretCheckLogin()
+	return c.secretsManager.SecretCheckLogin()
```

Comment:
When I was trying to use the secretsManager, I noticed that these functions would just keep recursively calling each other. This would cause another bug unless I'm misunderstanding how these work.